package/system/gpio-cdev/nu801/Makefile

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nu801
```

Comment:
This nested folder structure seems uncommon. Can you make it a package at the same level as all the others?
Or is this the beginning of a greater plan?

Reply:
@adschm TL;DR this is the first of it's kind, a userspace LED driver using CONFIG_GPIO_CDEV. Due to this, it felt appropriate to break it out into a new folder of gpio-cdev since more drivers like this may be added in the future. However, if you have a better idea on placement, I am open to suggestions.